CHANGES

```diff
@@ -9,7 +9,8 @@
 
  Changes between 1.1.1g and 1.1.1h [xx XXX xxxx]
 
-  *)
+  *) The Oracle Developer Studio compiler will start reporting deprecated APIs
+     unless configured with no-deprecated option.
```

Comment:
Ouch...  `no-deprecated` doesn't have quite the effect you think it has.  See INSTALL:

https://github.com/openssl/openssl/blob/315170f662b3053aa9fe817639e1b78f74f75077/INSTALL#L345-L348

In other words, with `no-deprecated`, using those deprecated functions will give you a compiler *error* because they are undefined.  If you want to get rid of the deprecation *warnings*, then you should give the compiler to turn off the deprecation warning when configuring.  With gcc, that would be `-Wno-deprecated-declarations`:

```
./config -Wno-deprecated-declarations
```

Reply:
I see, will remove that part.